LabTest1/src/Exercise1.java

```diff
@@ -0,0 +1,42 @@
+import java.math.BigInteger;
+import java.util.Scanner;
+
+public class Exercise1{
+	
+	static BigInteger factorial(int n) {

```

Comment:
you should have only main method and a public method that computes factorial between two numbers and call it to do all the job